experimenter/experimenter/theme/static_src/package.json

```diff
@@ -21,6 +22,11 @@
     "postcss-import": "^15.1.0",
     "postcss-nested": "^6.0.1",
     "postcss-simple-vars": "^7.0.1",
-    "tailwindcss": "^3.3.2"
+    "tailwindcss": "^3.3.2",
+    "webpack": "^5.88.2",
```

Comment:
Webpack is overkill for this. IMO we should use rollup cc @jaredlockhart

Reply:
What are the differences between Webpack and Rollup? (@Avi-88 you might be able to answer this :) )

Reply:
I haven't really worked with rollup but from what I learned while trying to use rollup is that it is primarily meant for bundling es modules and is much more simpler to use than webpack.

Reply:
Honestly I did struggle to get it to work right but I think I got it

Reply:
Rollup is supposed to be simpler to use and not require as much configuration as webpack when we just need to do some simple bundling, but that might not be the case here :(